xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -926,11 +926,13 @@ void CRenderManager::UpdateDisplayLatency()
   float refresh = fps;
   if (g_graphicsContext.GetVideoResolution() == RES_WINDOW)
     refresh = 0; // No idea about refresh rate when windowed, just get the default latency
-  m_displayLatency = (double) g_advancedSettings.GetDisplayLatency(refresh);
-
-  int buffers = g_Windowing.NoOfBuffers();
-  m_displayLatency += (buffers - 1) / fps;
+  m_displayLatency = g_advancedSettings.GetDisplayLatency(refresh);
```

Comment:
instead of changing the meaning of m_displaylatendy and introducing a new method TotalLatency() (just for debug purposes it stresses gfx context twice), you should update m_displayLatency in PrepareNextRender().  then the debug output can use m_displayLatency which is better anyway because you see the value that is actually used and not somthing updated at a different time